Meta/gn/build/buildflags.gni

```diff
@@ -0,0 +1,18 @@
+declare_args() {
+  # Build for debugging. Equivalent to is_optimized=false symbol_level=2.
+  is_debug = false
+}
+
+# args that depend on other args must live in a later declare_args() block.
+declare_args() {
+  # Whether to build with optimizations.
+  is_optimized = !is_debug
+
+  if (is_debug) {
+    # Debug info symbol level. 0: No symbols; 1: Line numbers; 2: Full symbols.
+    symbol_level = 2
+  } else {
+    # Debug info symbol level. 0: No symbols; 1: Line numbers; 2: Full symbols.
+    symbol_level = 0
+  }
```

Comment:
Does this mean that GN-built Ladybird does not have any debug info?

Reply:
Yes, unless you manipulate the gn args (such as `gn args out --args=symbol_level=1`) to get the symbol level to 1  or higher.

Reply:
Could that be accountable for some of the build speed increase? IIRC our CMake Lagom build unconditionally includes line tables (`-g1`).

Reply:
Some? Definitely not all of it. With `symbol_level=1` out/args.gn, I got this:

```
real	1m54.314s
user	18m19.242s
sys	1m40.234s
```

I'm personally convinced that the build speedup is organizing the jobs better, and doing less work around generated code. It's possible the missing c/cxxflags are causing the compiler to do less work for each cpp file though 🤷

Reply:
I've got local changes to default to symbol_level=1